contents/docs/feature-flags/early-access-feature-management.mdx

```diff
@@ -1,33 +1,54 @@
 ---
 
-Early access feature management enables your users the ability to opt-in to betas and other early features. This is especially useful if you don't want to run a beta application process, or if you ship frequent updates and want users to control their own experience. 
+Early access feature management enables your users to opt in (and out) of betas and other in-progress features. 
 
-Users can also decide to opt-out of betas they do not want to be part of. PostHog tracks opt-ins and opt-outs, so you can see how popular a beta is with your users. 
+This is useful if you want to:
+- Run a beta program without building a custom solution
+- Provide access to features that are not yet ready for general release
+- Allow users to control their own product experience
 
 ## How it works
 
-Early access features can be configured in the early access menu. Features are powered by feature flags, so each early access feature must be linked to an existing feature flag or a feature flag with the equivalent name to the feature will be implicitly created and linked.
+Early access features can be created and edited from the "[Early Access Management](https://app.posthog.com/early_access_features)" tab in PostHog.
+
+Features are powered by feature flags. Flags are automatically created when a feature is created (based on the name of the feature), or you can link an existing feature flag to a feature. Each feature can also contain a description and documentation link to help users understand what the feature is for.
 
 ![Create feature flags](../../images/features/feature-flags/create-feature-release.png)
 
-The default stage of an early access feature is beta. Users who opt-in to the feature will appear in the person list.
+Once created, you can opt users in or out manually on the feature detail screen by searching for them in PostHog. You can also implement a public opt-in flow which we detail below.
+
+> **Important:** Early access feature opt-in is an *overriding condition*. This means that a user opt-in or opt-out overrides any existing release condition on the feature flag. Only when a user has not explicitly selected to opt in or out will the release conditions on the feature flag be used to determine feature flag availability.
+
+## Creating a public opt-in
+
+> You can find a full walkthrough of building an app with early access feature management in our "[How to set up a public beta program using early access management](/tutorials/public-beta-program)" tutorial.
+
+### Option 1: Site app
 
-> Important: Early access feature opt-in is an *overriding condition*. This means that a user opt-in or opt-out will override any existing release condition on the feature flag. Only when a user has not explicitly selected to opt-in or opt-out will the release conditions on the feature flag be used to determine feature flag availability.
+We've prebuilt a site app that adds a user early access feature modal to your site. To use it:
```

Comment:
We should add a screenshot of what this looks like, so people can decide whether they'd like to use it or not

Reply:
Moved up the screenshot from further down